docs/user-guide/mvd-configuration.md

```diff
@@ -17,23 +17,22 @@ For earlier releases, you must register the Application Server with the Mediatio
 
 ### Accessing the Application Server
 
-**Note:** Before Accessing the Application Server, first [create and configure the Zowe instance directory](configure-instance-directory.md).
```

Comment:
Change not relevant here, we need to totally redo configure-instance-directory.md anyway